IntegrationTests/Services/AWSKinesisIntegrationTests/KinesisTests.swift

```diff
@@ -22,7 +23,10 @@ class KinesisTests: XCTestCase {
         // Resources will be cleaned up before the test concludes, pass or fail, unless the test crashes.
 
         let streamName = UUID().uuidString
-        let client = try KinesisClient(region: "us-west-2")
+        let client = try await KinesisClient
+            .builder()
+            .withPlugin(RegionPlugin("us-west-2"))
+            .build()
```

Comment:
We have to keep our client initializer non-async, at least for now.  This has caused problems with amplify-swift when we made client construction async in the past.

Reply:
The background here is that asynchronous Swift is a relatively recent thing, and not all Swift is asynchronous.

Almost all of the Swift SDK is async at this point, but there are a couple things like client constructors that we have agreed to keep synchronous for Amplify.